test/models/work_test.rb

```diff
@@ -0,0 +1,153 @@
+require "test_helper"
+
+describe Work do 
+  before do 
+    @album1 = works(:album1)
+    @album2 = works(:album2)
+    @album3 = works(:album3)
+    @user1 = users(:user1)
+    @user2 = users(:user2)
+    @user3 = users(:user3)
+  end
+
+  describe "validations" do
+    let (:new_album) {
+      Work.new(
+        category: "album",
+        title: "new album title",
+        creator: "new album creator",
+        publication_year: 2020,
+        description: "new album description"
+      )
+    }
+
+    it "it is valid when all fields are present" do
+      expect(@album1.valid?).must_equal true
+    end
+
+    it "will have the required fields" do
+      [:category, :title, :creator, :publication_year, :description].each do |field|
+        expect(@album1).must_respond_to field
+      end
+    end
+    
+    it "fails validation when there is a missing title" do
+      new_album.title = nil
+
+      expect(new_album.valid?).must_equal false
+      expect(new_album.errors.messages.include?(:title)).must_equal true
+      expect(new_album.errors.messages[:title].include?("can't be blank")).must_equal true
+    end
+    
+    it "fails validation when there is a missing category" do
+      new_album.category = nil
+
+      expect(new_album.valid?).must_equal false
+      expect(new_album.errors.messages.include?(:category)).must_equal true
+      expect(new_album.errors.messages[:category].include?("can't be blank")).must_equal true
+   end
+ 
+
+   it "fails validation when there is a missing creator" do
+      new_album.creator = nil
+
+      expect(new_album.valid?).must_equal false
+      expect(new_album.errors.messages.include?(:creator)).must_equal true
+      expect(new_album.errors.messages[:creator].include?("can't be blank")).must_equal true
+   end
+ 
+
+   it "fails validation when there is a missing description" do
```

Comment:
Minor little thing: There's some accidental extra indentation happening on several of these tests. I use a plugin called indent-rainbow to help me catch things like this.